### PR TITLE
chore: Spike for prototyping landing pages

### PR DESCRIPTION
**Pull request by abbyoung**
# SC-2599

Related PR: https://github.com/USSF-ORBIT/ussf-portal-cms/pull/384

## Proposed changes

This is a research branch for determining the scope of work that needs to be done to implement v1 landing pages. I've left a number of TODOs (there are more I am sure!), and there's a lot of scaffolding / placeholder.

We can possibly keep this branch open as a reference as we write stories + implement them in a more thorough fashion (like, with unit tests.)

### Start the system
```sh
yarn services:up
yarn dev
cd ../ussf-portal-cms
yarn dev
```

Login to the portal http://localhost:3000

### Start storybook

```sh
yarn storybook
```

Login to storybook http://localhost:6006, though the command above should open it for you

---

### As the original developer, I have

- [ ] Met the acceptance criteria
- [ ] Created new stories in Storybook if applicable
- [ ] Created/modified automated unit tests in Jest
- [ ] Created/modified automated [E2E tests](https://github.com/USSF-ORBIT/ussf-portal)
- [ ] Followed guidelines for zero-downtime deploys, if applicable
- [ ] Use [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) to check for basic a11y issues

### As a reviewer, I have

Check out our [How to review a pull request](https://github.com/USSF-ORBIT/ussf-portal/blob/main/docs/how-to/review-pull-request.md) document.

---

### CODE REVIEW

File: src/operations/cms/queries/getInternalNewsArticles.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const GET_INTERNAL_NEWS_ARTICLES = gql`
       where: {
         status: { equals: Published }
         publishedDate: { lte: $publishedDate }
-        category: { equals: InternalNews }
+        category: { name: { equals: "InternalNews" } }
```
Comment (abbyoung) [inline src/operations/cms/queries/getInternalNewsArticles.ts:9]: Changing category from an enum to a CMS field means this value has to change

File: src/pages/landing-pages/[landingPage].tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            {documents &&
+              documents.map((s: DocumentSectionType, i: number) => (
+                <Grid col={12} key={s.id} className={styles.accordionGrid}>
+                  <Accordion
+                    className={styles.accordion}
+                    bordered
+                    items={[
+                      {
+                        title: s.title,
+                        expanded: i === 0 ? true : false,
```
Comment (jcbcapps) [inline src/pages/landing-pages/[landingPage].tsx:48]: I'm wondering if we need to remove this line? Not sure if we want to automatically expand the first section here like we do on `/ussf-documentation`. Maybe a question for design?

File: src/pages/landing-pages/[landingPage].tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+              <Collection
+                key={`collection_${collection.id}`}
+                title={collection.title}
+                className={'sitesAndAppsCollection'}>
+                {/* TODO Fix type here */}
+                {/* eslint-disable-next-line @typescript-eslint/no-explicit-any */}
+                {collection.bookmarks?.map((bookmark: any) => (
+                  <Bookmark
+                    className={'sitesAndAppsBookmark'}
+                    key={`bookmark_${bookmark.id}`}
```
Comment (jcbcapps) [inline src/pages/landing-pages/[landingPage].tsx:90]: Just wanted to flag for later that both the Collection key and the Bookmark key are coming back undefined and printing to the console as a duplicate key error.

File: src/pages/landing-pages/[landingPage].tsx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+                          </div>
+                        ),
+                      },
+                    ]}
+                  />
+                </Grid>
+              ))}
+          </Grid>
+          <h2>Articles</h2>
+          <ArticleList articles={articles} />
```
Comment (jcbcapps) [inline src/pages/landing-pages/[landingPage].tsx:77]: Question: if the CMS user doesn't provide any articles (or if none exist), do we want to even render this section? Same question applies to all other sections on the page. Maybe another q for design.

**Review by jcbcapps — COMMENTED** [review]
Looking great so far! Just left a couple of questions/notes for later.

File: src/pages/landing-pages/[landingPage].tsx (hunk 2 above)
Comment (abbyoung) [reply src/pages/landing-pages/[landingPage].tsx:48]: agree -- i personally think all sections should be expanded on page load but that's just me 😄

**Pull request closed without merging**